src/app/common/rpc/use-rpc-sip30-broadcast-transaction.ts

```diff
@@ -50,7 +50,10 @@ export function useRpcSip30BroadcastTransaction(method: RpcMethodNames) {
   const { origin, requestId, tabId } = useRpcRequestParams();
   const txPayload = useLegacyTxPayloadFromRpcRequest();
   const stacksTransaction = useUnsignedStacksTransaction(txPayload);
-  const { stacksBroadcastTransaction } = useStacksBroadcastTransaction({ token: '' });
+  const { stacksBroadcastTransaction } = useStacksBroadcastTransaction({
+    token: '',
+    showSummaryPage: false,
```

Comment:
I had originally added `rpc-request` as an actionType to work with the hook, but somehow I removed passing it here. I removed it's use in the component now in favor of this, or we could keep ...neither are what we want, but I was following the already present `isCancelTransaction` and `isIncreaseFeeTransaction` in the hook.